CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+# 1.5.0-beta02 (August 2023)
+
+## Common
+
+### Features
+- [`androidx.compose.material.DropdownMenu` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/738)
+- [`androidx.compose.material3.DropdownMenu` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/737)
+- [`androidx.compose.material.AlertDialog` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/708)
+- [`androidx.compose.material3.AlertDialog` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/710)
+- [Add `PopupProperties.clippingEnabled` setting](https://github.com/JetBrains/compose-multiplatform-core/pull/740)
+- [Support `androidx.compose.material3.DatePicker`](https://github.com/JetBrains/compose-multiplatform-core/pull/717)
+- [Support `androidx.compose.material3.DatePickerDialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/745)
+
+### Fixes
+- [Fix sending synthetic event on detach owner without recomposition](https://github.com/JetBrains/compose-multiplatform-core/pull/730)
```

Comment:
It just allowed to remove workaround, no changes in public

Reply:
Fixed